README.md

```diff
@@ -55,9 +61,37 @@ sudo apt-get install git
 and then do:
 
 ```
+cd /home/$USER
 git clone https://github.com/anmavrid/webgme-bip.git
 ```
+This makes the 'project root' for the git repo `/home/$USER/webgme-bip` (you can pick a different location for it besides `/home/$USER/` if you'd like).
 
-Start mongodb locally by running the `mongod` executable in your mongodb installation (you may need to create a `data` directory or set `--dbpath`).
+Start mongodb locally by running the `mongod` executable in your mongodb installation (you may need to create a `data` directory or set `--dbpath`). For example:
+```
+cd /home/$USER
+mkdir bip_data
+mongodb --dbpath ./bip_data
+```
+wait until you see a line that says "[initandlisten] waiting for connections on port 27017".
 
-Then, run `webgme start` from the project root to start . Finally, navigate to `http://localhost:8888` to start using the WebGME-BIP design studio!
+Then, in a new terminal window, run `webgme start` from the project root (`webgme-bip`) to start. For example:
+```
+cd /home/$USER/webgme-bip
+webgme start
+```
+
+Finally, navigate to `http://localhost:8888` to start using the WebGME-BIP design studio!
+```
+firefox "https://127.0.0.1:8888/" &
+```
+
+As an aside... you can also download and install `webgme` itself locally if you'd like to play around with the basic (non-bip) interface:
+```
```

Comment:
You can do this from the bip repository. It contains webgme as a node_module and all "generic" visualization is included.